### DISCUSSION TITLE
No difference between read and not read articles

### DISCUSSION THREAD

**Discussion by shift18** [spacecowboy/Feeder#381] [General]
Is it just me or is it neigh on impossible to discern the difference between read and not read articles?
I mistakenly open articles all the time which I've already read.
Your thoughts, please!
![Screenshot_20241010-003846](https://github.com/user-attachments/assets/e23b5993-3a0f-4587-8d5a-e10af607cd86)

**Comment by KAGEYAM4** [comment]
Read articles Titles are unbolded.  In the screenshot, the article `2.6.33 released` is not read yet whereas `2.6.32 released` is read.


https://github.com/spacecowboy/Feeder/issues/379

**Reply by shift18** [reply]
True, just wish it was obvious whether they're read or not.

**Reply by shift18** [reply]
Didn't notice that issue, thanks for linking it.

**Comment by spacecowboy** [comment]
I hide read articles

**Reply by shift18** [reply]
Me too, but sometimes i need to check some read article so i enable them but then forget to disable them once I'm done.
Next time i open feeder I'm oblivious to the fact that read articles are enabled and i confuse myself as i don't notice the fact that read articles are displayed.
